pkgs/racket-doc/scribblings/reference/hashes.scrbl

```diff
@@ -570,26 +570,41 @@ with the following order (earlier bullets before later):
 @history[#:changed "6.3" @elem{Added the @racket[try-order?] argument.}
          #:changed "7.1.0.7" @elem{Added guarantees for @racket[try-order?].}]}
 
-@defproc[(hash-keys [hash hash?])
+@defproc[(hash-keys [hash hash?] [try-order? any/c #f])
          (listof any/c)]{
 Returns a list of the keys of @racket[hash] in an unspecified order.
 
-See @racket[hash-map] for information about modifying @racket[hash]
-during @racket[hash-keys]. @see-also-concurrency-caveat[]}
+If @racket[try-order?] is true, then the order of keys and values passed to
```

Comment:
It looks like this sentence came from the `hash-map` documentation, but there's no `proc` here. I think just dropping the sentence, leaving the next one (that directs a reader toward `hash-map`) is probably fine.